walletlibrary/src/main/java/com/microsoft/walletlibrary/mappings/presentation/CredentialPresentationInputDescriptorsMapping.kt

```diff
@@ -68,8 +67,8 @@ internal fun toVcPathRegexConstraint(fields: List<Fields>): VerifiedIdConstraint
     return GroupConstraint(vcPathRegexConstraints, GroupConstraintOperator.ALL)
 }
 
-internal fun toVcTypeConstraint(vcTypes: List<String>): VerifiedIdConstraint? {
-    if (vcTypes.isEmpty()) return null
+internal fun toVcTypeConstraint(vcTypes: List<String>): VerifiedIdConstraint {
+    if (vcTypes.isEmpty() || vcTypes.filterList { isNotBlank() }.isEmpty()) throw VcTypeConstraintsMissingException("There is no VerifiedId Type in credential input descriptor.")
```

Comment:
Would a developer or a user understand what a credential input descriptor is?

Reply:
yeah you are right, I will change that.